sdk/storage/azblob/zt_examples_test.go

```diff
@@ -338,7 +338,7 @@ func ExampleBlobAccessConditions() {
 	// This helper function displays the results of an operation; it is called frequently below.
 	showResult := func(response *DownloadResponse, err error) {
 		if err != nil {
-			if stgErr, ok := err.(StorageError); !ok {
+			if stgErr, ok := err.(*StorageError); !ok {
```

Comment:
Why is it a pointer now?

Reply:
Error() targets a pointer, and the SDK's generated code hands off a pointer to NewResponseError.

As a part of defunkification, we could hand back a non-pointer version.